docs/how_to/customization/custom_documents.md

```diff
@@ -1,17 +1,17 @@
 ## Customizing Documents
```

Comment:
doesn't have to this PR, but i'd love to update the docs to show usages of 
- excluded_llm_metadata_keys
-  excluded_embed_metadata_keys
- customizing the metadata formatting template

would be awesome to showcase to users how they can have more control over their data :)

Reply:
It will probably be this PR :) Will try to get an "advanced usage" notebook/docs page out

Reply:
Added some detailed docs for this!